scripts/wise.ts

```diff
@@ -45,7 +50,75 @@ program.command('check-expense <expenseId>').action(async expenseId => {
 
   const transfer = await transferwiseLib.getTransfer(connectedAccount, expense.data.transfer['id']);
   const recipient = await transferwiseLib.getRecipient(connectedAccount, transfer.targetAccount);
-  console.dir({ transfer, recipient }, { depth: null });
+  const quote = await transferwiseLib.getQuote(connectedAccount, transfer.quoteUuid);
+  const paymentOption = quote.paymentOptions.find(p => p.payIn === 'BALANCE' && p.payOut === quote.payOut);
+  console.dir({ transfer, recipient, quote, paymentOption }, { depth: null });
+
+  if (
+    'price' in expense.data.paymentOption && // Check if it is QuoteV3
+    paymentOption.price.priceDecisionReferenceId !== expense.data.paymentOption.price?.priceDecisionReferenceId
+  ) {
+    console.warn(
+      `Payment option mismatch! Reference stored in DB: ${expense.data.paymentOption.price?.priceDecisionReferenceId}, reference from Wise: ${paymentOption.price.priceDecisionReferenceId}`,
+    );
+  }
+  sequelize.close();
+});
+
+const checkPaymentProcessorFee = ({ expense, quote }) => {
+  const paymentOption = quote.paymentOptions.find(p => p.payIn === 'BALANCE' && p.payOut === quote.payOut);
+  assert.equal(expense.data.paymentOption.fee.total, paymentOption.fee.total, `Fee mismatch`);
+};
+
+program.command('check-host <hostSlug> [since]').action(async (hostSlug, since) => {
+  since = since || moment.utc().startOf('year');
+  console.log(`Checking expense for host ${hostSlug} since ${since}`);
+  const host = await models.Collective.findOne({ where: { slug: hostSlug } });
+  const [connectedAccount] = await host.getConnectedAccounts({
+    where: { service: Service.TRANSFERWISE, deletedAt: null },
+  });
+  if (!connectedAccount) {
+    return printAndExit(`${host.slug} not connected to Wise`, 'error');
+  }
+  const expenses = await models.Expense.findAll({
+    where: {
+      HostCollectiveId: host.id,
+      status: 'PAID',
+      updatedAt: { [Op.gte]: since },
+      data: { transfer: { [Op.ne]: null } },
+      feesPayer: 'COLLECTIVE',
```

Comment:
If I understand correctly:

```suggestion
      feesPayer: 'COLLECTIVE', // We don't care about other feesPayer as payees normally don't have a balance
```

In which case it would be worth double-checking.